.github/workflows/tests-integration.yml

```diff
@@ -0,0 +1,18 @@
+name: Run Integration Tests
+on:
+  workflow_call
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.17.0'
+
+      - name: Run Integration Tests
+        run: go test -tags=integration ./...
```

Comment:
Is the third dot in `...` correct?

Reply:
Yes. Apparently, `go test ./...` means "test all packages in a directory tree".